float64 position_x
float64 position_y
float64 orientation_yaw
